babel/messages/frontend.py

```diff
@@ -118,6 +118,12 @@ def finalize_options(self):
                                        'or the base directory')
 
     def run(self):
+        domains = self.domain.split()
+
+        for domain in domains:
+            self.__run_domain(domain)
+
+    def __run_domain(self, domain):
```

Comment:
Ah, please avoid double-underscore-prefixed names. Just `_run_domain` or `run_domain` would be fine!